src/main/java/com/taskadapter/redmineapi/internal/URIConfigurator.java

```diff
@@ -115,6 +116,14 @@ private URI createURI(String query,
         }
     }
 
+    static Collection<? extends RequestParam> distinct(Collection<? extends RequestParam> origParams) {
+        return origParams
+                .stream()
+                .filter(Objects::nonNull)
+                .collect(Collectors.toMap(RequestParam::getValue, a -> a, (s1, s2) -> s1))
```

Comment:
@alexeyOnGitHub like this, different
`RequestParam{name='f[]', value='status_id'`
`RequestParam{name='f[]', value='cf_13'` can be filtered, which is total need.
Still, if ever duplicate values occure, the map would skip one. Ideally the key/value pair should be checked.
But that's it for the moment.